server.go

```diff
@@ -1644,3 +1642,22 @@ func (s *server) Peers() []*peer {
 
 	return peers
 }
+
+// SetAlias updates the server's alias and broadcasts a new node announcement
+//
+// NOTE: This function is safe for concurrent access.
+func (s *server) SetAlias(alias string) error {
+	nodeAlias, err := lnwire.NewNodeAlias(alias)
+	if err != nil {
+		return err
+	}
+	s.currentNodeAnn.Alias = nodeAlias
+
+	signedNodeAnn, err := s.genNodeAnnouncement(true)
+	if err != nil {
+		return err
+	}
+	s.BroadcastMessage(nil, &signedNodeAnn)
```

Comment:
since we don't care about the result of this broadcast, we can maybe call it as a goroutine?